applications/NXazint1d.nxdl.xml

```diff
@@ -93,7 +94,31 @@
       </field>
 
       <field name="normalization_applied" type="NX_BOOLEAN">
-        <doc>is normalization correction applied or not.</doc>
+        <doc>
+          is normalization correction applied or not.
+
+          The normalization correction accounts for the effective number or weighted contribution of detector pixels to each integration bin.
+          An important aspect of the normalization strategy is how polarization and solid angle corrections are incorporated, which can vary
+          depending on the specific application, software, and its configuration options (see, for example, PyFAI documentation).
+          Additionally, the normalization strategy may include a relative or absolute calibration factor. Two common normalization approaches are:
+          "Relative normalization" to the PONI (Point Of Normal Incidence) pixel, and "Absolute calibration", which yields the number of photons
+          scattered by the sample in a given direction per unit solid angle.
```

Comment:
This is confusing for the casual reader as this flag does not express which normalization has been applied.